hw3/ThronesAPI/src/Searchbar.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import NavbarComponent from "./NavbarComponent";
+import { Container, Form, Row, Col, Button } from "react-bootstrap";
+
+const Searchbar = ({ characters, setSearchResults }) => {
+  const [searchCharacter, setSearchCharacter] = useState("");
+
+  const handleSearch = () => {
+    const matchedCharacters = characters.filter((character) =>
+      character.fullName.toLowerCase().includes(searchCharacter.toLowerCase())
+    );
+    setSearchResults(matchedCharacters);
+  };
+  return (
+    <>
+      <NavbarComponent />
+      <Container>
+        <Row>
+          <Col style={{ justifyContent: "center" }}>
+            <Form className="my-5 d-flex">
+              <Form.Control
```

Comment:
Interesting, you used a form for this. I didn't think to do that. Also, I don't know what Form.Control does but I will look into it after this.